src/ir_Corona.cpp

```diff
@@ -557,7 +557,7 @@ stdAc::state_t IRCoronaAc::toCommon(const stdAc::state_t *prev) {
   result.celsius = true;
   result.degrees = getTemp();
   result.fanspeed = toCommonFanSpeed(getFan());
-  if (getSwingV()) {
+  if (getSwingVToggle()) {
     result.swingv = (result.swingv != stdAc::swingv_t::kOff) ?
```

Comment:
The way I've tackled this in the past is to keep things simple everywhere else, and put the complexity into `handleToggles()`:
e.g.
https://github.com/crankyoldgit/IRremoteESP8266/blob/eb7cf83ed94099d311c6f30623332b1414406cd0/src/ir_Sharp.cpp#L646-L649
&
https://github.com/crankyoldgit/IRremoteESP8266/blob/eb7cf83ed94099d311c6f30623332b1414406cd0/src/IRac.cpp#L1377-L1383

Reply:
ok so if it is in `IRac::handleToggles` there is no need to handle the prev state in `IRCoronaAc::toCommon` ?

Reply:
That is/was the plan. That way we also de-dup some code. .. some. Not a lot.
Also makes "testing" simpler. ;-) Have I mentioned I'm lazy? :)

Reply:
Reading this, I think I copied logic from Coolix, which seems to toggle both in in `handleToggles` and in `toCommon` hmm, is that then maybe a bug?

Reply:
It's entirely possible.